lib/Sema/ImportResolution.cpp

```diff
@@ -397,6 +397,15 @@ ImportResolver::getModule(ImportPath::Module modulePath) {
     }
   }
 
+  // Only allow importing "Volatile" with Feature::Volatile or Feature::Embedded
```

Comment:
we should allow this in non-embedded no?

Reply:
yes -- it's allowed in the current implementation in this PR